internal/pkg/service/appsproxy/proxy/apphandler/authproxy/selector.go

```diff
@@ -101,11 +112,11 @@ func (s *Selector) writeSelectorPage(w http.ResponseWriter, req *http.Request, s
 	}
 
 	// Render the page, if there is no cookie or the value is invalid
-	s.pageWriter.WriteSelectorPage(w, req, status, s.selectorPageData(req, app, handlers))
+	s.pageWriter.WriteSelectorPage(w, req, status, s.selectorPageData(req))
 	return nil
 }
 
-func (s *Selector) selectorPageData(req *http.Request, app api.AppConfig, handlers map[provider.ID]*Handler) *pagewriter.SelectorPageData {
+func (s *SelectorForAppRule) selectorPageData(req *http.Request) *pagewriter.SelectorPageData {
 	// Pass link back to the current page, if reasonable, otherwise the user will be redirected to /
```

Comment:
I create `SelectorForAppRule` to reduce number of arguments and to make the code more clear.

Reply:
In my opinion it looks better than before. :+1: